ggml/src/ggml-cann/ggml-cann.cpp

```diff
@@ -470,23 +487,22 @@ std::unique_ptr<ggml_cann_pool> ggml_backend_cann_context::new_pool_for_device(
  */
 struct ggml_backend_cann_buffer_context {
     int32_t device;  ///< The device ID associated with this buffer context.
-    void* dev_ptr =
-        nullptr;  ///< Pointer to the device memory allocated for the buffer.
+    ggml_cann_pool_alloc* alloc;  ///< Pointer to the device memory allocated for the buffer.
```

Comment:
vmm requires stack memory application and release, that is, the memory is applied first and then released, otherwise an error will occur. Is it possible to guarantee such application and release order?

Reply:
As far as I know, tensor buffer and kv cache will use this buffer.